### PR TITLE
Update main.py

### PR DESCRIPTION
**Pull request by banillie**
attempt at fixing an issue in relation to the number of tabs (in wb) that bcompiler processes. Solution is a hard coded one. Not designed to be flexible. But, designed to address the immediate matter at hand.

### CODE REVIEW

File: bcompiler/compile.py [hunk 1]
```diff
@@ -60,12 +60,15 @@ def parse_source_cells(source_file: str, datamap_source_file: str) -> \
...
     datamap_obj.cell_map_from_csv(datamap_source_file)
     for item in datamap_obj.cell_map:
+        if item.template_sheet == "":
+            continue
         if item.template_sheet is not None and item.cell_reference is not None:
             try:
                 ws = wb[item.template_sheet]
             except KeyError as e:
                 logger.critical(f"{e}.{source_file} is not a BICC template. Not processing. Remove it!")
                 sys.exit()
```
Comment (hammerheadlemon) [inline bcompiler/compile.py]: You should maybe amend this to remove `sys.exit()` (which quits the program) and leave the CRITICAL log message  on line 69 which will flag a potentially catestrophic failure (trying to process a sheet that isn't there) but try to continue processing anyway.

**Comment by hammerheadlemon** [comment]
Can you think how you might be able to make your change generic, so that sheet names aren't hard-coded. Challenge!

**Comment by banillie** [comment]
I've started looking at this and have a good idea in my head how to do it. I'm struggling at the moment though setting-up git properly to create a new branch for the code which I can mess around with, without impacting the working code my computer. I've created another branch named challenge, by not sure the version control system is set up properly though. Need to do a lot more digging around/ learning git to understand the branching first, before I attempt to implement my change. key question is how can I revert to master after making/testing a load of changes in challenge.

**Comment by hammerheadlemon** [comment]
Sounds good Will. In Pycharm you can create and switch branches easilyhttps://www.jetbrains.com/help/pycharm/manage-branches.html. Just make sure you commit your changes to that branch before you switch back to master. Also remember that your PR is from your master. The preferred way to submit a PR is to do your work on a new branch and push that up to your repo and make a PR from that, but for this one, just keep it on master (is if you do succeed in doing more on your new branch, merge it all back to master before you update the PR with another push).

File: bcompiler/compile.py (hunk 1 above)
Comment (banillie) [reply bcompiler/compile.py]: from what I can tell the programme seems to be working fine without the extra change you suggest above. So up to you whether this change is necessary or not.

**Comment by banillie** [comment]
Hopefully you can see the changes that I have made. I've tested them on my computer and they are working fine! I hope the comments I've made in the code make sense? I probably need to send to you different masters, templates, dms and configs to test on your computer? Let me know what you think.

**Comment by hammerheadlemon** [comment]
Can you send me all the files I need in a zip, Will? (Master, template, datamap and config). Egress is fine. Cheers!

**Comment by banillie** [comment]
Have sent them to your matt lemon email account dude.

**Comment by hammerheadlemon** [comment]
@banillie Merged, with a bit of clean-up and published as version 1.3.20 on pypi (do `pip install -U bcompiler` to update your system copy of `bcompiler`). Congratulations!

To ensure your development copy is up to date with the latest version (ie the repo you're working with in Pycharm, you will need to update from upstream (i.e this `bcompiler` repo).

* Make sure you've got a clean working directory - ie you have not made any change to any files following your recent PR commit.
* Ensure you are on `master` branch.
* Open up Terminal in Pycharm.
* Check you have a clean working directory with `git status`. Ensure there are no files updated. If there are, do `git checkout -- .` to revert to a clean state.
* Do `git fetch upstream` to pull in the latest changes from this repo to your fork.
* Do `git pull upstream master`.

If everything is good, your `HEAD` should be at `32ad3af`, tagged `v1.3.20` - `applying fix to allow additional sheets to be parsed - PR#1 - 1.3.20`.

You're now good to hack away and submit your next PR!

**Pull request merged**